jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -26,7 +21,7 @@ Evaluations provide insights into the performance and efficiency of applications
 
 The term _evaluation_ initially referred to a way to rate and compare AI models. It has since expanded to include application-level testing, including Retrieval Augmented Generation (RAG), function calling, and agent-based applications.
 
-The evaluation process involves using a dataset of inputs to an LLM or application code, and a method to determine if the returned response matches the expected response.  There are many evaluation methodologies, such as:
+The evaluation process involves using a dataset of inputs to an LLM or application code, and a method to determine if the returned response matches the expected response. There are many evaluation methodologies, such as:
 
 * LLM-assisted evaluations
```

Comment:
I would like to add descriptions to go along with each of these but can always do that in a separate ticket

Reply:
I think that makes sense, my sense is we are eager to get this update out sooner than later!